examples/hello_world/.metadata

```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: 2d3ff10d62a1f1fdcbdbe92cbcc5a5594f95d500
+  channel: unknown
```

Comment:
Unknown seems strange here. Shouldn't that say master or dev?

Reply:
Yes, this should say master!

Reply:
I filed a ticket here:
https://github.com/flutter/flutter/issues/22833

Reply:
This is an artifact of creating the project from a fork of flutter, instead of a clone of the main repo. On a clone of the main repo, it says "master".